samples/drivers/led_gpio/src/main.c

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
As suggested above, it'd be better if we can fix the existing basic applications instead of creating a new one.

Reply:
I thought about this but using the LEDs GPIO drivers needs DTS modification for every board. So this PR is only a sample how to align the board dts. I can align all Nordic boards after it will be merged. Every LED have to have id field in its descriptions. So if I modify this sample then it will be broken for not aligned board. That is why I decided to create the new one sample. By the way this simple basic blinky sample in not broken. I think using LED driver is not mandatory for all boards so only those who decided to align their board could use it.